app/submitter/converter.py

```diff
@@ -104,6 +105,7 @@ def convert_answers(
 def build_collection(metadata) -> Mapping[str, str]:
     return {
         "exercise_sid": metadata["collection_exercise_sid"],
+        "instrument_id": metadata.get("form_type"),
         "schema_name": metadata["schema_name"],
         "period": metadata["period_id"],
     }
```

Comment:
This should only contain the `instrument_id` key if `form_type` exists, that is how the top-level form type is done.